src/main/java/newbank/client/ExampleClient.java

```diff
@@ -44,21 +54,44 @@ public void run() {
               String response = null;
               try {
                 response = bankServerIn.readLine();
+                gotReply = true;
```

Comment:
Jules, I know it's the most vague question ever. But how confident are you in merging it? :D Seems like a major change to the "core engine".

Reply:
No, not really. I would rely on our battery of tests :)

Reply:
Sounds good!